hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieCommitMetadata.java

```diff
@@ -75,6 +77,14 @@ public String getMetadata(String metaKey) {
     return extraMetadataMap.get(metaKey);
   }
 
+  public Boolean getCompactionCommit() {
```

Comment:
rename: isCompactionCommit() , per standards for boolean getter and setters

Reply:
This is auto-generated from getters and setters. The "is" is autmatically avoided.